### ISSUE TITLE
Logfile archiving message is outputted on every start

### ISSUE THREAD

**Issue by t-b** [AllenInstitute/MIES#2115]
The message

```
Just a moment, archiving log file XXX.
This is only done once.
```

from ArchiveLogFilesOnceAndKeepMonth() is shown on every startup.

I presume this is because we collect more than 512MiB in a logfile per month. But a sample logfile from TJ needs to verify that.

THis also needs a fix which does not complicate the archiving.

**Comment by t-b** [comment]
Needs a screenshare on the machine.

**Comment by t-b** [comment]
FTP: issue-2115

**Comment by t-b** [comment]
Should be fixed with 5d1ddb36e (Logs: Move log file archiving to IgorQuitHook, 2024-07-23).
